version.js

```diff
@@ -1,6 +1,49 @@
 const github = require('@actions/github')
 const semver = require('semver')
 
+const DEFAULT_VERSION = '0.0.0'
+
+async function getCommitsOnBranch(branch, config) {
+    const commits = new Set()
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
I stole this loop from the GitHub docs on how to do pagination: https://octokit.github.io/rest.js/v18#pagination.